dropbox-workitem/src/main/java/org/jbpm/process/workitem/dropbox/DownloadFileWorkitemHandler.java

```diff
@@ -39,13 +39,15 @@
 @Wid(widfile = "DropboxDownloadFileDefinitions.wid", name = "DropboxDownloadFile",
         displayName = "DropboxDownloadFile",
         defaultHandler = "mvel: new org.jbpm.process.workitem.dropbox.DownloadFileWorkitemHandler()",
+        documentation = "${artifactId}/index.html",
         parameters = {
                 @WidParameter(name = "DocumentPath")
         },
         results = {
                 @WidResult(name = "Document")
         },
         mavenDepends = {
+                @WidMavenDepends(group = "${groupId}", artifact = "${artifactId}", version = "${version}"),
                 @WidMavenDepends(group = "com.dropbox.core", artifact = "dropbox-core-sdk", version = "3.0.5")
```

Comment:
same here ... dropbox is dependency of dropbox-workitem and thus will be on classpath when this workitem module is added as dependency